Resources/config/mongodb.xml

```diff
@@ -56,6 +56,9 @@
     <services>
         <!-- defaults -->
         <service id="doctrine_mongodb.odm.cache" alias="doctrine_mongodb.odm.cache.array" />
+        <service id="Doctrine\Common\Persistence\ObjectManager" alias="doctrine_mongodb.odm.document_manager" public="false" />
+        <service id="%doctrine_mongodb.odm.document_manager.class%" alias="doctrine_mongodb.odm.document_manager" public="false" />
+        <service id="Doctrine\Common\Persistence\ManagerRegistry" alias="doctrine_mongodb" public="false" />
```

Comment:
This will also conflict with ORM - you can use `Doctrine\Bundle\MongoDBBundle\ManagerRegistry`.